### PR TITLE
Header changes

### PR DESCRIPTION
**Pull request by Arskiainen**
Header and navigation styles. Missing DropdownLink and LanguageSelector tweaks, because those are pending HDS PRs or alternate solutions.

### CODE REVIEW

**Comment by Arskiainen** [comment]
We will make a separate PR for the DropdownLinks when/if they are available.

File: src/components/navigation/Header.tsx [hunk 1]
```diff
@@ -75,9 +86,24 @@ function Header(header:NavProps): JSX.Element {
...
       titleAriaLabel={t('navigation.title_aria_label')}
+      className={classNames(styles.navigation, styles.zover)}
     >
+      <Navigation.Row>
+        {getNavi(menu)}
+      </Navigation.Row>
       <Navigation.Actions>
-        <Navigation.Row variant='inline' key='languages'>
+        <Navigation.Search searchLabel="Search" searchPlaceholder="Search page" />
+        <Navigation.User
```
Comment (Jussiles) [inline src/components/navigation/Header.tsx:96]: Should these "Search" strings be translated?

**Review by Jussiles — APPROVED** [review]
Looks nice! Just on comment about translations.

File: src/components/navigation/Header.tsx (hunk 1 above)
Comment (Arskiainen) [reply src/components/navigation/Header.tsx:96]: Placeholder, we don't have search yet so this is just for documentation now.

**Pull request merged**